exporters/prometheus/src/exporter_utils.cc

```diff
@@ -220,6 +271,51 @@ prometheus_client::MetricType PrometheusExporterUtils::TranslateType(
   }
 }
 
+void PrometheusExporterUtils::SetTarget(const sdk::metrics::ResourceMetrics &data,
+                                        std::vector<::prometheus::MetricFamily> *output)
+{
+  if (output == nullptr || data.resource_ == nullptr)
+  {
+    return;
+  }
+  if (data.scope_metric_data_.empty())
+  {
+    return;
+  }
+  if ((*data.scope_metric_data_.begin()).metric_data_.empty())
+  {
+    return;
+  }
+
+  prometheus_client::MetricFamily metric_family;
+  metric_family.name = "target_info";
```

Comment:
this should just be `target`.  See https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#supporting-target-metadata-in-both-push-based-and-pull-based-systems. I'll update the prometheus specification to fix this.
```suggestion
  metric_family.name = "target";
```

https://github.com/open-telemetry/opentelemetry-specification/issues/3700